src/main/resources/reference.conf

```diff
@@ -170,7 +173,7 @@ resourceTypes = {
       project-owner = {
         # This role is used for display purposes and to prevent owners from removing project-owners,
         # but otherwise grants the same actions as the owner role on workspaces.
-        roleActions = []
+        roleActions = ["share_policy::project-owner"]
```

Comment:
Do we need/want to add this? Any project owner in Terra would be able to update the project owners with this change, but I thought we wanted to limit it to Rawls.